sinonjs/sinon
# Remove obsoleted lib/sinon/sandbox-stub.js

This should have been removed in 74263dd9c5154587e1436d1b742af6d0137bd399,
where we removed a lot of deprecated things. This file was overlooked and
was left behind.


#### To verify

1. Check out this branch
1. `ag -Q "sandbox-stub"`
1. Observe that there are no references found
